### ISSUE TITLE
Don't hardcode executable paths

### ISSUE THREAD

**Issue by lukasepple** [grindhold/rainbow-lollipop#3]

**Comment by lukasepple** [comment]
```
Using default config

** (rainbow-lollipop:24055): CRITICAL **: rainbow_lollipop_application_get_state: assertion 'self != NULL' failed

** (rainbow-lollipop:24055): CRITICAL **: rainbow_lollipop_application_get_state: assertion 'self != NULL' failed

(rainbow-lollipop:24055): Clutter-CRITICAL **: clutter_actor_get_last_child: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:24055): CRITICAL **: rainbow_lollipop_track_recalculate_y: assertion 'self != NULL' failed


(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed

** (rainbow-lollipop:24055): CRITICAL **: rainbow_lollipop_node_get_splits: assertion 'self != NULL' failed

(rainbow-lollipop:24055): Clutter-CRITICAL **: clutter_actor_get_last_child: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:24055): CRITICAL **: rainbow_lollipop_track_recalculate_y: assertion 'self != NULL' failed

(rainbow-lollipop:24055): Clutter-CRITICAL **: clutter_actor_get_x: assertion 'CLUTTER_IS_ACTOR (self)' failed

(rainbow-lollipop:24055): Clutter-CRITICAL **: clutter_actor_get_width: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:24055): CRITICAL **: rainbow_lollipop_node_recalculate_y: assertion 'self != NULL' failed

(rainbow-lollipop:24055): Clutter-CRITICAL **: clutter_actor_get_last_child: assertion 'CLUTTER_IS_ACTOR (self)' failed

** (rainbow-lollipop:24055): CRITICAL **: rainbow_lollipop_track_recalculate_y: assertion 'self != NULL' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_height: assertion 'GTK_IS_WIDGET (widget)' failed

(rainbow-lollipop:24055): Gtk-CRITICAL **: gtk_widget_get_preferred_width: assertion 'GTK_IS_WIDGET (widget)' failed
```

**Comment by grindhold** [comment]
can you somehow reproduce this issue?

**Comment by lukasepple** [comment]
When I open the browser I am in address insert mode. It can be toggled
with TAB just fine, but as soon as I enter an address and load it I
can't switch to address insert mode anymore. (This issue occurs
irrelevant if I restore or start a new session)

**Comment by lukasepple** [comment]
Okay, installing it into /usr/local fixed it.

**Comment by grindhold** [comment]
should be fixed with 0134f6f
@lukasepple  could you please check?

**Comment by lukasepple** [comment]
Works fine now.
